bin/ansible_tower-operations

```diff
@@ -44,6 +45,15 @@ SourcesApiClient.configure do |config|
   config.host   = "#{args[:sources_host]}:#{args[:sources_port]}"
 end
 
+ActiveSupport::Notifications.subscribe(/.*heartbeat\.consumer\.kafka.*$/) do |*options|
```

Comment:
If the Kafka notifications is emited once per 10 seconds and heartbeat timeout is also 10 seconds, it can cause problems in edge cases

Reply:
I increased the limit for checking heartbeat file to 12 seconds so it should be enough